pkg/kubernetes/scripts/projects.js

```diff
@@ -406,6 +408,24 @@
                 return false;
             }
 
+            function makeRegistryAdmin(project, currentUser) {
+                var subject = {
+                    kind: 'User',
+                    name: toName(currentUser),
+                };
+                //if system:admin return
+                if(currentUser.groups.indexOf("system:cluster-admins")!= -1) {
+                    var deferred = $q.defer();
+                    deferred.resolve();
+                    return deferred.promise;
```

Comment:
What are we resolving here. You can just use ```$q.when({ });``` or something like that, instead of these three lines.